### PR TITLE
add index disk size fields

### PR DESCRIPTION
**Pull request by nginthfs**

### CODE REVIEW

File: solrman/solrmanapi/cloudstatus.go [hunk 1]
```diff
@@ -20,12 +20,14 @@ import "time"
...
-	DiskSize      float64
-	IndexDiskSize float64
+	Hostname               string                     // the node's hostname, as determined by gethostbyaddr
+	NodeName               string                     // the node's identifier within SolrCloud (e.g. "1.1.1.1:8983_solr")
+	Cores                  map[string]*SolrCoreStatus // keys are core names
+	Zone                   string                     // cloud zone where host exists
+	DiskSize               float64
+	IndexDiskSize          float64
+	IndexDiskSizeTotal     int64
+	IndexDiskSizeAvailable int64
```
Comment (hiteshk25) [inline solrman/solrmanapi/cloudstatus.go:30]: Just curious why we need `IndexDiskSizeAvailable` ?

**Review by hiteshk25 — APPROVED** [review]
LGTM

File: solrman/solrmanapi/cloudstatus.go (hunk 1 above)
Comment (nginthfs) [reply solrman/solrmanapi/cloudstatus.go:30]: We need both total and available so we can tell what percentage full the disk is. We need `IndexDiskSizeAvailable` too so we can calculate if moving a shard onto the disk will cause it to go over the 90% threshold.

Comment (hiteshk25) [reply solrman/solrmanapi/cloudstatus.go:30]: i thought we get raw `DiskSize` from gcp-computeInsatnce-api`, and we just need used space.

**Pull request merged**